src/styles/timeline.less

```diff
@@ -1,3 +1,9 @@
+ux-timeline {
```

Comment:
Currently every Angular component has it's stylesheet in the same folder as the template and the component class. It might be good to create the template for this component in there. I think this would be a good idea as when we go to remove the AngularJS version we can then simply delete this stylesheet as the Angular one will not be dependant on it.

Reply:
New timeline.component.less stylesheet created. References to ng1 styles have been removed. Changes made in the PR to src/styles/timeline.ess have been reverted with the exception of changing the arrow icon from 'hpe-down' to 'hpe-chevron-down' and adding a transformation of 0.5px for the chevron.